backend/src/main/java/de/janka/capstonedrazen/controller/FlatsController.java

```diff
@@ -32,14 +30,20 @@ public FlatsController(FlatService flatService) {
     }
 
     @PostMapping("/publish")
-    public ResponseEntity<Flat> create(@AuthenticationPrincipal UserEntity authUser, @RequestBody Flat flat){
+    public ResponseEntity<Flat> create(@RequestBody Flat flat){
 
         FlatEntity createdFlatEntity = flatService.create(flat);
         Flat createdFlat = map(createdFlatEntity);
 
         return ok(createdFlat);
```

Comment:
kannst einenmap über findBy suchen ob die Wohnujg schon in der Datenbank existiert, falls nein -> Wohnung erstellen, falls ja -> Exception werfen 

Dtos kann man auch einbauen.

Alles was jetzt kommt sind erstmal Vorschläge, darfst du umsetzen wie du lustig bist^^

Reply:
Danke Tobi 👍